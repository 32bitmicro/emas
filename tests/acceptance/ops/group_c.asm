RIC r0
ZLB r0
SXU r0
NGA r0
SLZ r0
SLY r0
SLX r0
SRY r0
NGL r0
RPC r0
SHC r0, 0
RKY r0
ZRB r0
SXL r0
NGC r0
SVZ r0
SVY r0
SVX r0
SRX r0
SRZ r0
LPC r0
