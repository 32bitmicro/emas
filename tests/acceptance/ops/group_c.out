@ 0x0000 : 0xe800  /  111 010 0 000 000 000  /  59392
@ 0x0001 : 0xe801  /  111 010 0 000 000 001  /  59393
@ 0x0002 : 0xe802  /  111 010 0 000 000 010  /  59394
@ 0x0003 : 0xe803  /  111 010 0 000 000 011  /  59395
@ 0x0004 : 0xe804  /  111 010 0 000 000 100  /  59396
@ 0x0005 : 0xe805  /  111 010 0 000 000 101  /  59397
@ 0x0006 : 0xe806  /  111 010 0 000 000 110  /  59398
@ 0x0007 : 0xe807  /  111 010 0 000 000 111  /  59399
@ 0x0008 : 0xe808  /  111 010 0 000 001 000  /  59400
@ 0x0009 : 0xe809  /  111 010 0 000 001 001  /  59401
@ 0x000a : 0xe810  /  111 010 0 000 010 000  /  59408
@ 0x000b : 0xea00  /  111 010 1 000 000 000  /  59904
@ 0x000c : 0xea01  /  111 010 1 000 000 001  /  59905
@ 0x000d : 0xea02  /  111 010 1 000 000 010  /  59906
@ 0x000e : 0xea03  /  111 010 1 000 000 011  /  59907
@ 0x000f : 0xea04  /  111 010 1 000 000 100  /  59908
@ 0x0010 : 0xea05  /  111 010 1 000 000 101  /  59909
@ 0x0011 : 0xea06  /  111 010 1 000 000 110  /  59910
@ 0x0012 : 0xea07  /  111 010 1 000 000 111  /  59911
@ 0x0013 : 0xea08  /  111 010 1 000 001 000  /  59912
@ 0x0014 : 0xea09  /  111 010 1 000 001 001  /  59913
